src/main/java/com/kitaab/hisaab/ledger/controller/ReminderController.java

```diff
@@ -0,0 +1,58 @@
+package com.kitaab.hisaab.ledger.controller;
+
+
+import com.kitaab.hisaab.ledger.dto.request.reminder.SendReminderRequest;
+import com.kitaab.hisaab.ledger.dto.response.SuccessResponse;
+import com.kitaab.hisaab.ledger.entity.Reminder;
+import com.kitaab.hisaab.ledger.repository.UserRepository;
+import com.kitaab.hisaab.ledger.service.ReminderService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("api/users/")
+public class ReminderController {
+
+    @Autowired
+    private ReminderService reminderService;
+
+    @Autowired
+    UserRepository userRepository;
+
+    @GetMapping("/sent")
+    public Page<Reminder> showSentReminders(@RequestParam(defaultValue = "1", required = false) int pageNo,
+                                            @RequestParam(defaultValue = "10", required = false) int pageSize,
+                                            @RequestParam(required = true) String userId) {
+        Optional.of(userRepository.findById(userId)).get()
```

Comment:
`userRepository.findById` by default returns a `Optional<T>`. It is not required to wrap that again inside the `Optional`